### PR TITLE
add website contributing link

### PR DESCRIPTION
**Pull request by xmulligan**
Signed-off-by: Bill Mulligan <billmulligan516@gmail.com>

Please ensure your pull request adheres to the following guidelines:

- [ ] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [ ] All code is covered by unit and/or runtime tests where feasible.
- [ ] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [ ] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [ ] Provide a title or release-note blurb suitable for the release notes.
- [ ] Thanks for contributing!

add a link to the website contributing section

### CODE REVIEW

**Review by pchaigno — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
+## To Cilium/Hubble
+
 See the [Developer / Contributor
 Guide](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/) for detailed information on
 how to contribute, get started and find good first issues.
+
+## To the website
+
+Please see the [website contributing guide](https://github.com/cilium/cilium.io/blob/main/CONTRIBUTING.md) for detailed
+information on how to add blogs, trainings, and other resources
```
Comment (kaworu) [inline CONTRIBUTING.md]: nitpicking:
```suggestion
## To the cilium.io website

Please see the [cilium.io website contributing guide](https://github.com/cilium/cilium.io/blob/main/CONTRIBUTING.md) for detailed
information on how to add blogs, trainings, and other resources.
```

**Review by kaworu — CHANGES_REQUESTED** [review]
Thanks for the PR @xmulligan! Blocking this PR until https://github.com/cilium/cilium.io/pull/11 is merged, but other than that LGTM.

**Comment by kaworu** [comment]
Thanks @xmulligan! Please squash the two commits into one.

**Review by pchaigno — CHANGES_REQUESTED** [review]
We need a proper title for the commit (i.e., not `squash`). (Alex suggested to squash the commits, but the exact action probably should have been `fixup`.) The previous commit title should be okay.

**Review by kaworu — APPROVED** [review]

**Review by aditighag — APPROVED** [review]

**Pull request merged**